<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassifiersSettingsView</class>
 <widget class="QWidget" name="ClassifiersSettingsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <spacer name="m_qVerticalSpacer_Margin">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="m_qGridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="m_qLabel_TriggerThreshold">
       <property name="text">
        <string>Trigger Threshold:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="m_qSpinBox_TriggerThreshold">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="m_qLabel_Classifiers">
       <property name="text">
        <string>Classifiers:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="m_qComboBox_Classifiers"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="m_qLabel_TriggerClass">
       <property name="text">
        <string>Trigger Class</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="m_qComboBox_TriggerClass"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="m_qHorizontalSpacer_Margin">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
